wiki/Front-end_with_curl.md

```diff
@@ -0,0 +1,278 @@
+# Curl Examples to front-end
+
+The front-end is accessable not only via the web GUI.
+
+<h3>Introduktion</h3>
+<p>
+The front-end has a bridge functionality that is built in as a part to the front-ends web server and handles request coming from the web GUI towards any of the back-ends configured.
+The front-end may also be used by other applications such as CURL or any kind of program that can make HTTP(S) requests.
+</p>
+
+<h3>The front-end handels:</h3>
+<ul>
+   <li>POST, PUT and GET requests</li>
+   <li>Default back-end</li>
+   <li>Input back-end</li>
+</ul>
+<p>
+While the web gui may use the back-end instances list and select different back-ends in an easy way, a user that uses the front-end without the included web GUI may need to specify a back-end URL.
+This may be done by adding a backendurl parameter to the request.
+</p>
+<p>
+Note that for users where the front-end and back-end is runnign with the help of Tomcat there will be context paths used.
+</p>
+<h3>Default back-end</h3>
+<p>
+The front-end may have been setup mwith a default back-end, this means that if no back-end is specified when making a HTTP request to the front-end the default back-end will be used.
+If you want to see a list of back-ends and see if there is a default back-end set you may use the command:
+</p>
+<pre>
+Normal curl command for get:
+curl -X GET http://*front-end-url*/*contect-path-if-any*/backend
+Curl command for get with host, port using Tomcat:
+curl -X GET http://localhost:8080/eifrontend/backend
+Curl command running the .war file:
+curl -X GET http://localhost:8080/backend
+</pre>
+<p>
+The default back-end should have the key <code>defaultBackend</code> set to <code>true</code>. If the JSON list ends up empty there is no back-ends specified in the front-end. If there is no JSON object with the key set to true there is no default back-end.
```

Comment:
"If the JSON list ends up empty there ARE ..."

Reply:
Done.